src/App.js

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-import { connect } from 'react-redux';
-
-import NavigationContainer from './nav/NavigationContainer';
-
-const App = (props) =>
-  <NavigationContainer />;
-
-const mapStateToProps = (state) => ({
-  isLoggedIn: state.app.isLoggedIn,
```

Comment:
Do we need an `isLoggedIn` state for it, or could we just check the validity of the api key in the `restore` function?